lib/Module/Starter/Simple.pm

```diff
@@ -440,6 +440,7 @@ my %WriteMakefileArgs = (
         #'ABC'              => '1.6',
         #'Foo::Bar::Module' => '5.0401',
     },
+    macro => { TARFLAGS => '--format=ustar -c -v -f' },
     dist  => { COMPRESS => 'gzip -9f', SUFFIX => 'gz', },
```

Comment:
According to [ExtUtils::MakeMaker](https://metacpan.org/pod/ExtUtils::MakeMaker#dist) documentation, `TARFLAGS` should be in `dist` key, not in `macro` key.

Reply:
Interesting, it seems to work both ways. Also, I'm not sure what I originally read about BSD, it seems its tar supports `--format ustar` as well as `-o`.

Reply:
The `dist` section is options for creating the distribution tarball which is what we want here. `macro` is undocumented so we would need an archaeologist to find its purpose.

Reply:
`macro` is general purpose, `dist` is for building tarballs specifically. `macro` happens to work because it happens to be defined after dist, but this shouldn't be relied upon.